### ISSUE TITLE
Sprint 26 a11y support

### ISSUE THREAD

**Issue by LynneJohnson224** [department-of-veterans-affairs/va.gov-team#77081]
## User Story
As a shared support team member, I need to understand a11y needs each MHV product team has so I can ensure we align with design system requirements and provide the best user experience possible for Veterans and other users of VA.gov. 

## Description
Any MHV teams including: Secure Messaging, Medical Records, Medications, and also related Appointments & Check-In teams may need to request shared support services for accessibility. Within a given sprint, that work should be articulated through tasks (below) and updates (comments on the ticket) as needed. 

### Ongoing tasks: 
- [ ] 

### New tasks this sprint
- [x] Going through MHV tools in staging environment to review state of accessibility. During this review I'm finding bugs and will communicate these to Bobby Bailey to see if tickets need to be made. 
- [x] Create a survey form to send out to teams that gauges interest in Office Hours and workshops.
